src/containers/views/DraftEdit.js

```diff
@@ -157,6 +195,13 @@ export class DraftEdit extends Component {
                 block
               />
               <Splitter />
+              <Button
+                onClick={() => this.handleClickPublish(path)}
+                type="publish"
+                icon="send-o"
+                active={true}
```

Comment:
No need for `true`. Just pass `active`.

Reply:
This was in sync with existing JSX on `master`:
https://github.com/jekyll/jekyll-admin/blob/a592eafc18611eae46a0182a9cae9f203854b69e/src/containers/views/DraftEdit.js#L152-L166

I'll send a PR to fix `master` first..